backend/models/bcf.js

```diff
@@ -408,816 +464,639 @@ function getBCFMarkup(issue, account, model, unit) {
 	});
 }
 
-bcf.getBCFZipReadStream = function(account, model, username, branch, revId, ids, useIssueNumbers = false) {
-
+bcf.getBCFZipReadStream = function(account, model, issues, unit) {
```

Comment:
if there isn't much overlap between read/write, it might be good to just split them into 2 files.

Reply:
I think BCF can be kept as one file?